fp.js

```diff
@@ -1,14 +1,16 @@
 // Convert the following code to use `.map`
-
 var peoplesAges = []
+
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
-  peoplesAges.push(person.age)
-})
+
+// function peopleMap() {
+//   var peoplesAges = people.map(age);
+// }
+
```

Comment:
close, the first argument that you pass the callback to `.map` is a variable representing **each** item in the array. i.e.

```js
// es6
people.map( person => person.age )

// es5
people.map( function (person) { return person.age })
```